app/src/main/kotlin/com/wire/android/core/extension/Activity.kt

```diff
@@ -10,4 +12,9 @@ fun AppCompatActivity.replaceFragment(frameId: Int, fragment: Fragment, addToBac
                 it.addToBackStack(fragment.tag)
             }
         }
-    }
\ No newline at end of file
+    }
+
+fun Activity.showKeyboard() {
```

Comment:
I'm a big fan of inlining in order to be more concise, except when it makes the code more difficult to read, but here is how it would look like:

```kotlin
fun Activity.showKeyboard()  = 
      (this.getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager).toggleSoftInput(SHOW_IMPLICIT, HIDE_IMPLICIT_ONLY)
```

Then you need to import: ```InputMethodManager```